modules/hello-world/pages/start-using-ottoman.adoc

```diff
@@ -109,8 +116,40 @@ async function main() {
 }
 ----
 
-Now, let's create a connection to our Couchbase Server.
+Now, let's create a connection to our database.
+
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+To connect to Couchbase Capella, be sure to set up credentials for your database by xref:cloud:get-started:cluster-and-data.adoc#credentials[following these steps].
+Capella requires mandatory use of TLS (Transport Layer Security), therefore a security certificate must be provided when connecting.
 
+If you don't already have your security certificate, you can download it from the Capella UI:
+
+. Go to the "Clusters" section, in the left hand navigation.
+. Click on the relevant cluster.
+. Select the "Connect" tab and scroll down to the "Security Certificate" section.
+. Download the "Root Certificate".
+
+[source,javascript]
+----
+include::hello-world:example$start-using-ottoman-cloud.js[tag=connect]
+----
+
+TIP: If your application is running in a xref:ref:client-settings.adoc[constrained network environment] (i.e., your laptop), you may experience timeout issues.
```

Comment:
Just trying to help users with this, after the confusion on slack:
https://couchbase.slack.com/archives/CC6GX1CJJ/p1661970119229539

@RichardSmedley , let me know if this seems correct for Capella, or if needs tweaking.